<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_material</class>
 <widget class="QDialog" name="Edit_material">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>711</width>
     <height>61</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AllEditTriggers</set>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Tên vật liệu</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nhiệt luyện</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kích thước</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Độ rắn</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Giới hạn bền</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Giới hạn chảy</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Công thức ƯSTX</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qml.qrc">
        <normaloff>:/dowload/Anh_cua_app+do_an/Edit.png</normaloff>:/dowload/Anh_cua_app+do_an/Edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Chỉnh sửa vật liệu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>168</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>121</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>2.8 x σch</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40 x HRCm</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Toi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Toi cai thien</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tham Cacbon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tham Nito</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Toi the tich</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
